jax/_src/xla_bridge.py

```diff
@@ -246,6 +246,52 @@ def make_cpu_client() -> xla_client.Client:
     "cpu", make_cpu_client, priority=0, fail_quietly=False
 )
 
+def _check_ptxas_compat() -> None:
+  import re
+  import jaxlib
+  import subprocess
+
+  from glob import glob
+
+  _jaxlib_path = pathlib.Path(jaxlib.__file__).parent
+  searchable_paths = [_jaxlib_path.parent / "nvidia" / "cuda_nvcc"]
+  searchable_paths += [pathlib.Path(path) for path in os.environ["PATH"].split(":")[:18]]
```

Comment:
This might not work on Windows; I think it uses a `;` as the separator.

We don't really support CUDA on windows, but it's probably worth at least trying to make it work.